lib/provider/volume_mounter.go

```diff
@@ -100,4 +105,6 @@ type VolumeDetachRequest struct {
 type VolumeAttachRequest struct {
 	Opts     map[string]string `json:"opts"`
 	HostName string            `json:"hostName"`
```

Comment:
Not used, please remove

Reply:
will remove 
	Opts     map[string]string `json:"opts"`
	HostName string            `json:"hostName"`